Source/JavaScriptCore/heap/Heap.h

```diff
@@ -1073,6 +1071,18 @@ class Heap {
     CompleteSubspace variableSizedCellSpace; // FIXME: This space is problematic because we have things in here like DirectArguments and ScopedArguments; those should be split into JSValueOOB cells and JSValueStrict auxiliaries. https://bugs.webkit.org/show_bug.cgi?id=182858
     CompleteSubspace destructibleObjectSpace;
```

Comment:
Why not using destructibleObjectSpace instead of creating webAssemblyInstanceSpace newly?